setup.py

```diff
@@ -17,10 +17,10 @@
 
 import setuptools
 
-setuptools.setup(name='confighelper',
-    description='Parkbench configuration parser',
-    author='Joel Allen Luellwitz and Andrew Klapp',
-    author_email='eviljoel@linux.com, andklapp@null.net',
-    license='GPLv3',
-    packages=['confighelper'],
-    zipsafe=False)
+setuptools.setup(name='parkbenchcommon',
+                 description='Miscellaneous Parkbench utilities',
+                 author='Joel Allen Luellwitz and Andrew Klapp',
+                 author_email='eviljoel@linux.com, andklapp@null.net',
+                 license='GPLv3',
+                 packages=['parkbenchcommon'],
```

Comment:
Does this include sub-packages?

Reply:
Those are called modules, and this package is a collection of modules, with no sub-packages.